frontend/src/metabase/dashboard/components/DashboardParameterPanel/DashboardParameterPanel.tsx

```diff
@@ -42,48 +41,63 @@ export function DashboardParameterPanel({
   const hasVisibleParameters = visibleParameters.length > 0;
   const shouldRenderAsNightMode = isNightMode && isFullscreen;
 
+  const allowSticky = isParametersWidgetContainersSticky(
+    visibleParameters.length,
+  );
+  const {
+    isSticky,
+    isStickyStateChanging,
+    intersectionObserverTargetRef: stickyRef,
+  } = useIsParameterPanelSticky();
+
+  const shouldApplyThemeChangeTransition = !isStickyStateChanging && isSticky;
```

Comment:
In both places (regular dashboard and EmbedFrame) we want to apply transitions to the panel ONLY when it is sticky and a theme is changing.

When a panel is non-sticky, it has a transparent background
When it becomes sticky and we add the `transition` property to it (that should affect theme change), we still don't want to perform a transition from the `transparent` bg to an actual bg color of a panel.
To achieve it before the `isSticky` state is changing we set `isStickyStateChangin` to `true` and just after it in a `requestAnimationFrame` change it back to `false`. 

Visually, it looks ok-ish, but we have 2 re-renders instead of 1 re-render every time when the `isSticky` value is changing.